examples/chat/module/rust/src/lib.rs

```diff
@@ -150,7 +150,9 @@ impl oak::CommandHandler for Router {
             _ => {
                 anyhow::bail!("received malformed gRPC invocation");
             }
-        }
+        };
+        command.close();
```

Comment:
Is it intentional to ignore errors due to closing channels (here and elsewhere)? does it not work to just use `?` for both, in a linear fashion?